lib/studies/reply/inbox.dart

```diff
@@ -479,21 +575,24 @@ class __BuildMobileNavState extends State<_BuildMobileNav>
                         _drawerController.reverse();
                         _dropArrowController.forward();
                         Future.delayed(
-                            Duration(
-                              milliseconds:
-                                  GalleryOptions.of(context).timeDilation == 1
-                                      ? _drawerController.value == 1 ? 350 : 210
-                                      : _drawerController.value == 1
-                                          ? 1750
-                                          : 1050,
-                            ), () {
-                          //Wait until animations are complete to reload the
-                          //state. Delay is variable based on if the gallery
-                          //is in slow motion mode or not.
-                          widget.onItemTapped(
-                              _destinationsWithIndex[destination]);
-                          _currentDestination = destination;
-                        });
+                          Duration(
+                            milliseconds:
+                                GalleryOptions.of(context).timeDilation == 1
```

Comment:
Hmm is there a reason you have to read the `timeDilation`? I think when slow motion is turned on, all animations automatically slow down without you having to adjust their duration. In fact I think by doing this the animation would just be even slower

Reply:
The reason I read timeDilation is because, the time that this delayed code should be run is based on how fast the animation is. Since timeDilation is multiplying the time by a factor of 5, I also need to change the time that the function is delayed by a factor of 5 when in slow motion, or else the code will run before desired and cut into the animation.

Reply:
`Future.delay()` is delaying a call to setState() because if I allow it to call setState() before the delay it causes a widget rebuild in the middle of the animation causing it to cut out.

Reply:
oh that makes sense then, my bad I didn't full read the surrounding code.